src/dom-wrapper-error.ts

```diff
@@ -0,0 +1,58 @@
+import { FindComponentSelector, DOMWrapperAPI } from './types'
+
+interface Options {
+  selector: FindComponentSelector
+}
+
+export class DOMWrapperError implements DOMWrapperAPI {
```

Comment:
nit: add some doc on when this class is created? I'm starting to get lost between VueWrapper, VuerapperAPI, DOMWrapper, ErrorWrapper, DOmWrapperAPi 😅

Reply:
Yes we definitely need to update docs. The fact we are in alpha and kind of changing this a lot is kind of the reason docs are not updated - I think once we get to beta we can then start to lock down the API and really get a handle on docs (VTU for Vue 2 were not great, we should def spend a good amount of time on this).